### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN053.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#103] [General]
# https://blog.3place.cn/3place/lmyn/LMYN053.html

https://blog.3place.cn/3place/lmyn/LMYN053.html

**Comment by picklecai** [comment]
美食博主，如意神牛

**Comment by picklecai** [comment]
我送给你十万头牛，
请把神牛舍波罗给我。
因为，世尊呀！它是宝贝，

吃饭吃出事来了。美食博主要被抢了。。

**Comment by picklecai** [comment]
我送给你十万头牛，
请把神牛舍波罗给我。
因为，世尊呀！它是宝贝，
国王正要保护舍波罗。
因此，把舍波罗给我吧！
婆罗门呀！这合乎达磨。’ （1.52.9）

见色起意/见宝起意，居然还自称“合乎达磨”。众友从前做国王，也是强盗逻辑呀。

**Comment by picklecai** [comment]
有了它我才能喊娑婆诃〔405〕、婆舍〔406〕，
娑婆诃〔405〕：古代印度宗教认为这是有神秘力量的声音。
婆舍〔406〕：祭祀时呼喊的声音。

**Comment by picklecai** [comment]
它是我的新月祭和满月祭，

新月要祭祀，满月要祭祀。
是不是天天都在祭祀？

**Comment by picklecai** [comment]
甘蔗、蜜和烘烤的粮食，
还有喝的烧酒和甜酒，
又有非常精美的饮料，
还有各样食品香甜可口。 （1.52.2）

滚热精美的大米饭，
堆得像那山岳一般，
洁纯的食品和汤，
稠稠的酸奶像河川。 （1.52.3）

有各种各样的甜食，
还有水果做成的糖果，
装满了糖制的盘子，
成百盘成千盘地摆着。 （1.52.4）

古印度美食：
甘蔗、蜜、烤饼、米饭、甜食、水果糖；
烧酒、甜酒、饮料、酸奶、汤；

**Comment by picklecai** [comment]
我给你黄金制造的车子，
这样的车子给你成百辆，
前面还套上四匹白马，
身上装饰着小小的铃铛。 （1.52.18）

四匹白马。古印度也用得上“驷马难追”。
